css/myModal.css

```diff
@@ -0,0 +1,44 @@
+/* The Modal (background) */
+.myModal {
+    display: none; 
+    color: White;
+    position: fixed; /* Stay in place */
+    z-index: 1; /* Sit on top */
+    left: 0;
+    top: 0;
+    width: 100%; /* Full width */
+    height: 100%; /* Full height */
+    overflow: auto; /* Enable scroll if needed */
+    background-color: rgb(0,0,0); /* Fallback color */
+    background-color: rgba(0,0,0,0.4); /* Black w/ opacity */
+  }
+  
+  /* Modal Content/Box */
+  .myModal-content {
+    position: relative;
+    margin: auto;
+    border-radius: 1.5vh;
+    background-color: #212121;
+    margin: 5% auto; /* 15% from the top and centered */
+    padding: 20px;
+    border: 1px solid #888;
+    width: 50%; /* Could be more or less, depending on screen size */
+  }
+  
+  /* The Close Button */
```

Comment:
Do we use that?
Do we need to close modal when we promote pawn?
What if someone close modal without promotion?